cvat/apps/engine/views.py

```diff
@@ -360,20 +372,133 @@ def get_permissions(self):
 
         return [perm() for perm in permissions]
 
-    def perform_create(self, serializer):
-        def validate_task_limit(owner):
-            admin_perm = auth.AdminRolePermission()
-            is_admin = admin_perm.has_permission(self.request, self)
-            if not is_admin and settings.RESTRICTIONS['task_limit'] is not None and \
-                Task.objects.filter(owner=owner).count() >= settings.RESTRICTIONS['task_limit']:
-                raise serializers.ValidationError('The user has the maximum number of tasks')
+    def _validate_task_limit(self, owner):
+        admin_perm = auth.AdminRolePermission()
+        is_admin = admin_perm.has_permission(self.request, self)
+        if not is_admin and settings.RESTRICTIONS['task_limit'] is not None and \
+            Task.objects.filter(owner=owner).count() >= settings.RESTRICTIONS['task_limit']:
+            raise serializers.ValidationError('The user has the maximum number of tasks')
+
+    def create(self, request):
+        action = self.request.query_params.get('action', None)
+        if action is None:
+            return super().create(request)
+        elif action == 'import':
+            self._validate_task_limit(owner=self.request.user)
+            if 'rq_id' in request.data:
+                rq_id = request.data['rq_id']
+            else:
+                rq_id = "{}@/api/v1/tasks/{}?action_import".format(request.user, uuid.uuid4())
```

Comment:
Why uuid here? Dataset export and annotations import/export don't use uuid.

Reply:
OK. The task doesn't exist here. You are  trying to use something here. It is fine.